app/src/main/kotlin/cz/covid19cz/erouska/exposurenotifications/ExposureNotificationsRepository.kt

```diff
@@ -34,6 +35,15 @@ class ExposureNotificationsRepository(
         return btAdapter.isEnabled
     }
 
+    fun isLocationEnabled(): Boolean {
```

Comment:
Co takhle použít `LocationManagerCompat` ? myslím, že je to aktuálně spolehlivější řešní
```
private fun isLocationEnabled(context: Context): Boolean {
    val locationManager = context.getSystemService(Context.LOCATION_SERVICE) as LocationManager
    return LocationManagerCompat.isLocationEnabled(locationManager)
}
```